### PR TITLE
Add PoC parsing of conformance test values encoded in ion

### PR DESCRIPTION
**Pull request by jpschorr**
Add PoC parsing of conformance test values encoded in ion

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: partiql-conformance-tests/tests/test_value.rs [hunk 1]
```diff
@@ -0,0 +1,231 @@
+use ion_rs::external::bigdecimal::BigDecimal;
```
Comment (alancai98) [inline partiql-conformance-tests/tests/test_value.rs:1]: nit: unused import

File: partiql-conformance-tests/tests/test_value.rs [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+use ion_rs::{Integer, IonReader, IonType, Reader, StreamItem};
+use partiql_value::{Bag, List, Tuple, Value};
+use std::fs::read;
```
Comment (alancai98) [inline partiql-conformance-tests/tests/test_value.rs:4]: nit: unused import

File: partiql-conformance-tests/tests/test_value.rs [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+use std::str::FromStr;
```
Comment (alancai98) [inline partiql-conformance-tests/tests/test_value.rs:5]: nit: unused import

File: partiql-conformance-tests/tests/test_value.rs [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+                f: 1,
+                d: 2e0,
+                s: 1
+            }
+        ]";
+        let expected = Value::from(partiql_bag![partiql_tuple![
+            ("f", 1),
+            ("d", Value::Real(2.0.into())),
+            ("s", 1)
+        ]]);
```
Comment (alancai98) [inline partiql-conformance-tests/tests/test_value.rs:158]: No assertion here currently. Should we add a call to `parse` with `test` and `expected`?

File: partiql-conformance-tests/tests/test_value.rs [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+                d: 2e0,
+                s: 1
+            }
+        ]";
+        let expected = Value::from(partiql_list![partiql_tuple![
+            ("f", 1),
+            ("d", Value::Real(2.0.into())),
+            ("s", 1)
+        ]]);
+    }
```
Comment (alancai98) [inline partiql-conformance-tests/tests/test_value.rs:230]: Also here, no assertion. Perhaps need to call with `parse`?

**Review by alancai98 — COMMENTED** [review]
Some minor comments (unused imports, test assertions). Parsing of the Ion values into PartiQL `Value`s looks good otherwise.

**Review by alancai98 — APPROVED** [review]

**Review by am357 — APPROVED** [review]

**Pull request merged**